src/main/java/org/prgrms/kdt/forward/IOController.java

```diff
@@ -0,0 +1,77 @@
+package org.prgrms.kdt.forward;
+
+import org.prgrms.kdt.controller.VoucherController;
+import org.prgrms.kdt.controller.request.CreateVoucherRequest;
+import org.prgrms.kdt.controller.response.VoucherResponse;
+import org.prgrms.kdt.forward.io.Input;
+import org.prgrms.kdt.forward.io.Output;
+import org.prgrms.kdt.view.ConsoleView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class IOController {
+    private final Input input;
+    private final Output output;
+    private final VoucherController voucherController;
+    private final ConsoleView consoleView;
+    private static final Logger logger = LoggerFactory.getLogger(IOController.class);
+
+    @Autowired
+    public IOController(Input input, Output output, VoucherController voucherController, ConsoleView consoleView) {
+        this.input = input;
+        this.output = output;
+        this.voucherController = voucherController;
+        this.consoleView = consoleView;
+    }
+
+    public void run() {
+        boolean exit = false;
+        logger.info("Start : Voucher Manage Program");
+        output.write(consoleView.introduceCommand());
+
+        do {
+            try {
+                String command = input.readLine();
+                switch (command) {
+                    case "create" -> {
+                        logger.info("Request <create voucher>");
+                        output.write(consoleView.requestVoucherInfo());
+                        String requestedVoucherInfo = input.readLine();
+                        CreateVoucherRequest createVoucherRequest = new CreateVoucherRequest(requestedVoucherInfo);
+                        if (voucherController.create(createVoucherRequest))
+                            output.write(consoleView.saveVoucher());
+                        else output.write(consoleView.saveVoucherError());
```

Comment:
```suggestion
	                    createVoucher();
```

create 라는 코드 라인이 많아 보이는데 이를 개선할 방법은 없는지 고민해보는 것도 좋을 것 같습니다.
mac을 사용하신다면 코드를 드래그 한 뒤에 option + command + m 을 누르시면 메서드로 추출이 가능합니다.
그 뒤에 해당 로직만 바라보면서 수정할만한 내용을 찾아보시죠.!

Reply:
흐음 당최 수정할 부분이 뭔지 모르겠네요..!
혹시 `output.write()`가 조건문에 중복 사용 된 것이 문제일까요?
아니면 error를 반환하는데, `output.error()`와 같은 메소드를 사용하지 않은 것이 수정할 부분일까요?